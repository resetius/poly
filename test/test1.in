+ + + * 2 ^ x 2 * x y ^ y 3 6
