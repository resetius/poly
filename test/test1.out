0,0,6
0,3,1
1,1,1
2,0,2
